pkg/store/bucket.go

```diff
@@ -716,6 +719,25 @@ func blockSeries(
 			return s.lset[i].Name < s.lset[j].Name
 		})
 
+		validTombstones := make([]tombstone.Tombstone, 0)
```

Comment:
If we don't allocate a specific amount, then we can simply do 
```suggestion
		var validTombstones []tombstone.Tombstone
```